sts/security-sts-server/src/main/java/ddf/security/sts/PropertyPlaceholderWrapper.java

```diff
@@ -45,10 +43,15 @@ public PropertyPlaceholderWrapper(DefaultConditionsProvider conditionsProvider,
             StaticSTSProperties properties) {
         samlConditionsProvider = conditionsProvider;
         stsProperties = properties;
+        // set the default values in case there is no configuration
+        samlConditionsProvider.setLifetime(1800);
+        stsProperties.setSignatureUsername("localhost");
+        stsProperties.setIssuer("localhost");
+        stsProperties.setEncryptionUsername("localhost");
```

Comment:
I could refactor them into constants for the class. Unfortunately this is how it was with the property-placeholder,  essentially. There's not really anywhere that I've found where these default values could be pulled from as constants.

Stylistically I could go either way, I'm fine with constants too.